src/main/java/fruitshop/service/reporthandlers/RepotrToFIleConverter.java

```diff
@@ -0,0 +1,28 @@
+package fruitshop.service.reporthandlers;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class RepotrToFIleConverter implements ReportSupplier {
```

Comment:
погана назва інтерфейсу і класу, переконайся що в тебе всі класи і інтерфейси названі по такому принципу
```suggestion
public class FileWriterImpl implements FileWriter {
```